geomesa-plugin/src/main/resources/GeoServerApplication.properties

```diff
@@ -52,3 +52,12 @@ HadoopStatusPage.label.main=Hadoop Cluster
 # binary viewer format labels
 format.wfs.application/vnd.binary-viewer=Binary Viewer
 format.wfs.bin=Binary Viewer
+
+GeoMesaCoverageStoreEditPanel.instanceId=Instance ID
+GeoMesaCoverageStoreEditPanel.user=Accumulo User
+GeoMesaCoverageStoreEditPanel.password=Accumulo Password
+GeoMesaCoverageStoreEditPanel.zookeepers=Zookeepers String
+GeoMesaCoverageStoreEditPanel.auths=Authorizations
+GeoMesaCoverageStoreEditPanel.visibilities=Visibilities
+GeoMesaCoverageStoreEditPanel.tableName=Accumulo Table Name
```

Comment:
Is this still a single-table system, or does this refer to the name of a catalog-esque table?